documentation/Migration/Update-1.0.0-rc.md

```diff
@@ -97,6 +97,36 @@ catch (UnsupportedDataSourceException)
 }
 ````
 
+## Interfaces implemented in the SDK or SDK.Runtime
+
+There are some changes made to interfaces that generally don't need to be implemented by plugins. Those changes are documented in this section.
+
+### ICustomDataProcessor
+
+- Adds `TryEnableTable` method. This is similar to `EnableTable`, but catches most exceptions and returns a `bool`.
+
+### IDataProcessorExtensibilitySupport
+
+- The `AddTable` method becomes `EnableTable` or `TryEnableTable`.
+
+- The `GetAllRequiredSourceDataCookers` was renamed to `GetRequiredSourceDataCookers`.
+
+- The `GetAllRequiredTables was renamed to `GetEnabledInternalTables`.
+
+### IDataExtensionRetrievalFactory
+
+This was moved from the SDK to SDK.Runtime.
+
+## TableAttribute and InternalTableAttribute
+
+- The `InternalTableAttribute` was removed.
+
+- Internal table constructor arguments and properties have been removed from `TableAttribute`.
```

Comment:
Doesn't https://github.com/microsoft/microsoft-performance-toolkit-sdk/pull/117/files do this? Can we just get that PR in to reduce the number of changes this PR is making?

Reply:
Actually, this change is a bit different. This removes internal from the table attribute all together. Rather than putting it on the table attribute, it is stored in the table descriptor. The table descriptor is what is stored longterm and should fully describe the table. I'm not sure how the table attributes and table descriptors evolved over time, but this is the correct place for it today.